features/shop/cart_with_different_taxes_for_different_zones.feature

```diff
@@ -0,0 +1,50 @@
+@ui-cart
+Feature: Cart with different tax rates for different time zones
+    In order to order products with different tax rate for different zones
+    As a Customer
```

Comment:
We should decide about the roles in the first place:

* Visitor (not logged in)
* Customer (logged in)
* Guest Customer (not logged in, but in checkout)
* Admin

Thoughts?